### ISSUE TITLE
Questions won't pop up???

### ISSUE THREAD

**Issue by Acotto561** [accimeesterlin/trivia-game#1]
// Initial Values
let counter = 30;
let currentQuestion = 0;
let score = 0;
let lost = 0;
let timer;

// Display questions an Choices in the broswer

function loadQuestion() {
  
  var question = quizQuestions[currentQuestion].question; //
  var choices = quizQuestions[currentQuestion].choices; //
  
  $("#game").html("<h4>" + question + "</h4>");
  
}

loadQuestion(); *this code does not pop up questions*

**Comment by accimeesterlin** [comment]
Can you also share your index.html code as well?

**Comment by Acotto561** [comment]
<!DOCTYPE html>

<html>
  <head>
    <meta charset="UTF-8"/>
    <meta name="viewport" contents="width=device-width, initial-scale=1.o"/>
    <meta http-equiv = "X-UA Compatible" content= "ie=edge"/>
    <title>Trivia Game</title>
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"/>
<link rel="stylesheet" href="style.css"/>
  </head>
  <body>
    
    <div class="container">
      
      <div class="jumbotron">
        <h1>Trivia Game</h1>
        <button class="btn btn-primary" id="start">Start Game</button>
      </div>
      
      <h2 id= "time"></h2>
    <div class="game"></div>
      
      </div>
    
    
    
    
    
    
    
  <script
  src="https://code.jquery.com/jquery-3.4.1.min.js"
  integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
  crossorigin="anonymous"></script>
   <script src="question.js"></script>
   <script src="app.js"></script>

  </body>
  
</html>

**Comment by accimeesterlin** [comment]
Can you also share the question.js code as well?

**Comment by Acotto561** [comment]
Yes my apologies! I’ll send everything right away thank you for the help!

**Comment by Acotto561** [comment]
var quizQuestions = [
  {
    question:"What is my name?",
    choices: ["Anthony","Ray","Juni","Bob"],
    correctAnswer:"Anthony"
  },

  {
    question:"Who is my favorite NFL team?",
    choices: ["Bucs","Pats","Fins","Jags"],
    correctAnswer:"Pats"
  },

  {
    question:"What City did you live in?",
    choices: ["West Palm","Miami","Sarasoda","Cali"],
    correctAnswer:"West Palm"
  }

]; //this is my question.js

**Comment by Acotto561** [comment]
// Initial Values
let counter = 30;
let currentQuestion = 0;
let score = 0;
let lost = 0;
let timer;

// Display questions an Choices in the broswer

function loadQuestion() {
  
  var question = quizQuestions[currentQuestion].question;
  var choices = quizQuestions[currentQuestion].choices;
  
  $("#game").html("<h4>" + question + "</h4>");
  
}

loadQuestion(); //app.js

**Comment by Acotto561** [comment]
so i put this in the console to see what it said...... $ ("#game").html ("<h4>"+question+"</h4>"); 
VM27:1 Uncaught ReferenceError: question is not defined
    at <anonymous>:1:26

**Comment by accimeesterlin** [comment]
Try this
$("#game").html(question);

I also notice that inside your index.html. You don't have an id named *game*, instead of you have a class. 

Instead of 
```
<div class="game"></div>
```


Try
```
<div id="game"></div>
```

**Comment by Acotto561** [comment]
this is what i had to do so the question can show   <button class="btn btn-primary" id="start" onclick=loadQuestion()>Start Game</button>

**Comment by Acotto561** [comment]
// <button class="btn btn-primary" id="start" onclick=loadQuestion() >Start Game<  /button>

**Comment by Acotto561** [comment]
I had to put a onclick in the button tag to get the question to show.... but the choices wont show. neither did the timer... I have no clue where im going wrong because i am following you...

**Comment by accimeesterlin** [comment]
Hey @Acotto561 ,
I have 40 minutes available. Can you join me on this video call?
Zoom ID: https://zoom.us/j/339789217

This zoom meeting ID will be unavailable after the meeting.
